src/commands/keys.ts

```diff
@@ -40,6 +40,22 @@ command(
   }
 )
 
+command(
+  'key-raw',
```

Comment:
The naming is a big weird, but most of the commands in the tool already have a bit of a "yoda" feel to them, so this name makes sense in context.